Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -7613,6 +9603,30 @@ def main():
         elif demisto.command() == 'panorama-install-file-content-update':
             panorama_install_file_content_update_command(args)
 
+        elif demisto.command() == 'pan-os-platform-get-arp-tables':
+            topology = get_topology()
+            result = get_arp_tables(topology, **demisto.args())
+            dataclasses_to_command_results(result, empty_result_message="No ARP entries.")
+        elif demisto.command() == 'pan-os-platform-get-route-summary':
+            topology = get_topology()
+            result = get_route_summaries(topology, **demisto.args())
+            dataclasses_to_command_results(result, empty_result_message="Empty route summary result.")
+        elif demisto.command() == 'pan-os-platform-get-routes':
+            topology = get_topology()
+            result = get_routes(topology, **demisto.args())
+            dataclasses_to_command_results(result, empty_result_message="Empty route summary result.")
+        elif demisto.command() == 'pan-os-platform-get-system-info':
+            topology = get_topology()
+            result = get_system_info(topology, **demisto.args())
+            dataclasses_to_command_results(result)
+        elif demisto.command() == 'pan-os-platform-get-device-groups':
+            topology = get_topology()
+            result = get_device_groups(topology, **demisto.args())
+            dataclasses_to_command_results(result, empty_result_message="No device groups found.")
```

Comment:
```suggestion
            dataclasses_to_command_results(get_device_groups(topology, **demisto.args()), empty_result_message="No device groups found.")
```

Reply:
please change that to all the commands

Reply:
I changed this to wrap dataclasses_to_command_results with return_results instead, I think this is the most readable. let me know.